### PR TITLE
image: Add shellcheck to the image

### PR DESCRIPTION
**Pull request by lyarwood**
/cc @0xFelix 

**What this PR does / why we need it**:

This needs to be a separate step in order for our post submit jobs to trigger and build the new image. This then needs to be used by our pre-submit job before adding the use of shellcheck to the lint script.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by lyarwood** [comment]
/hold

Just need to wait for https://github.com/kubevirt/project-infra/pull/2432 before we merge this.

File: image/Dockerfile [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 FROM quay.io/fedora/fedora-minimal:36
 
-RUN microdnf install -y make git golang python3-pip tar && microdnf clean all -y
+RUN microdnf install -y make git golang python3-pip tar xz && microdnf clean all -y
```
Comment (0xFelix) [inline image/Dockerfile:19]: Better install shellcheck with dnf, keep the case sensitive pkg name in mind (`ShellCheck`)

**Pull request merged**